.evergreen/config.yml

```
@@ -574,7 +574,7 @@ buildvariants:
 # Tests all PHP versions on all operating systems.
 # Only tests against latest MongoDB and ext-mongodb versions
 - matrix_name: "test-php-versions"
-  matrix_spec: {"os-php7": "*", "php-versions": "*", "edge-versions": "latest-stable", "driver-versions": "latest-stable" }
+  matrix_spec: {"os-php7": "*", "php-versions": "*", "edge-versions": "latest-stable", "driver-versions": "latest-dev" }
```

I added todo comments in the lines where I changed the `matrix_spec` to ensure we're changing it back everywhere. Happy to skip this in favour of reverting this commit if the changes are isolated.

---

This will likely get squashed so I'll leave todo comments.

```suggestion
  # TODO: restore driver-versions to latest-stable after 1.11 is tagged
  matrix_spec: {"os-php7": "*", "php-versions": "*", "edge-versions": "latest-stable", "driver-versions": "latest-dev" }
```